todo_app/tests/app_test.py

```diff
@@ -0,0 +1,113 @@
+    requests_mock.get('https://api.trello.com/1/search', text=board_data)
+    requests_mock.get('https://api.trello.com/1/boards/5f58c9c553d5e50e89e24311/lists', text=listinboard_data)
+    requests_mock.get('https://api.trello.com/1/boards/5f58c9c553d5e50e89e24311/cards', text=cardsinboard_data)
+    
+    assert 'Book my holiday' in client.get('/').data.decode("utf-8") 
+
+# def test_cant_add_number_to_string():
```

Comment:
All this commented out code can go (you won't lose it as it's in your source control history)